tests/Rules.Framework.IntegrationTests/Rules.Framework.IntegrationTests.csproj

```diff
@@ -25,7 +25,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\..\src\Rules.Framework\Rules.Framework.csproj" />
+    <ProjectReference Include="..\..\src\Rules.Framework.Providers.InMemory\Rules.Framework.Providers.InMemory.csproj" />
```

Comment:
This test project is supposed to only test the core package Rules.Framework. If you are changing/adding tests here for the in-memory provider, they should be moved to the in-memory integration tests project.

Reply:
Done

Reply:
Thank you Carlos 👌